### ISSUE TITLE
Link Styling

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1354]
**Component block:** none, global styling

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=123-520&t=8zhGPBubooE9KYeN-0

As a site visitor, I want clear state change(s) on hyperlinks so that I can quickly identify a hyperlink no matter how I access the website.

**Requirements:** 
Update default link styling to primary brand color
Links should include a visual state change on hover: links should include an underline on hover
Links should include a visual state change on focus: add background color on focus
Links should include a state change on press: text color changes + underline is retained on click
Links should include a disabled state
**Links should be white text on primary color background (example: alert block)
Links should be keyboard navigable
Links pass color contrast for AA
Links should be able to be read by a screen reader

**Acceptance Criteria:** 
1. Given I have added a hyperlink to plain text on my page, when the page renders, then the link should be styled with the primary brand text color. 
2. Given I am on a page with a text hyperlink, when I hover the hyperlink, then the link includes an underline.
3. Given I am on a page with a text hyperlink, when I focus the link via keyboard navigation, then the link includes a partial opacity background color. 
4. Given I am on a page with a text hyperlink, when I click the hyperlink, then the text color + underline change to a darker color (need to confirm darkest ramp color or black) to indicate it was clicked. 
5. Given I am on a page with a text hyperlink, when the hyperlink is disabled for any reason, then the hyperlink displays with the 400 neutral ramp color and underline.
6. Given I am on a page with a primary orange color block, when there is a hyperlink in the text present, then it is styled as a white link.
7. Given I am navigating a web page, when I click tab on my keyboard, then I am able to successfully focus on the text hyperlink in logical page order and see the focus state. 
8. Given I am viewing a web page with hyperlink text, when I use a color contrast checker, then the links on page pass AA compliance in any link state. 
9. Given I am viewing a web page with hyperlink text, when I am using a screen reader, then the reader is able to announce the hyperlink in context with all required screen reader aria tags

**Comment by sarah0arnold** [comment]
Hyperlinks reviewed in UAT session with project team 8/4/25 and passes with no feedback. Closing ticket
